_data/simple-icons.json

```diff
@@ -9274,6 +9274,11 @@
             "source": "https://www.make.com/en/press",
             "guidelines": "https://www.make.com/en/press"
         },
+        {
+            "title": "Makedeb",
+            "hex": "000000",
+            "source": "https://github.com/makedeb/brand"
```

Comment:
The linter is failing because for GitHub links you need a permalink to the specific file you used as source (https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines).